src/main/java/lotto/dto/LottoNumberResponse.java

```diff
@@ -0,0 +1,14 @@
+package lotto.dto;
+
+import lotto.domain.lottery.Lotto;
+
+import java.util.List;
+
+public record LottoNumberResponse(
+        List<Integer> numbers
+) {
+    public static LottoNumberResponse build(final Lotto lotto) {
+        List<Integer> numbers = lotto.getNumbers();
+        return new LottoNumberResponse(numbers);
```

Comment:
[크게 중요하지 않아요!]
```suggestion
        return new LottoNumberResponse(lotto.getNumbers());
```
이미 로또에서 숫자를 가져오는 결과를 메서드 명칭으로 쉽게 이해할 수 있었습니다! 그렇다면 변수로 추출않아도 좋을 것 같습니다!